symfit/core/fit_results.py

```diff
@@ -147,4 +191,73 @@ def _array_safe_dict_eq(one_dict, other_dict):
         else: return True
 
     def __eq__(self, other):
-        return FitResults._array_safe_dict_eq(self.__dict__, other.__dict__)
\ No newline at end of file
+        return FitResults._array_safe_dict_eq(self.__dict__, other.__dict__)
+
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        state['minimizer'] = (type(state['minimizer']),
+                              state['minimizer'].objective,
+                              state['minimizer'].parameters)
```

Comment:
Does this include parameter initial values?

Reply:
That question is work in progress ;). Still, the equality checking in the pickle testing passes successfully, so I'm going to say yes?